docs/build/configure/webhooks.md

```diff
@@ -39,7 +39,14 @@ To configure a webhook:
         "attributes": {
           "data_types": ["binary", "tabular", "file"]
         },
-        "type": "part_data_ingested"
+        "type": "part_data_ingested",
+        "headers": {
+          "Component-Type": "<Component type>",
+          "Component-Name": "<Component name>",
+          "Method-Name": "<Method name>",
+          "Min-Time-Received": "<Minimum time>",
+          "Max-Time-Received": "<Maximum time>",
```

Comment:
Do [these two](https://github.com/viamrobotics/app/blob/2df56ed1f0a95a3f45088580e0cfd10009bc02e2/webhooks/utils.go#L34-L37) need to be added to the `"headers"` data objects on page as well? Otherwise, LGTM!

Reply:
Gotcha! Thanks for looking into it. We do have a few (methods | variables | settings) that are only to be used for internal tests, and indeed shouldn't be documented.

@nataliajacobowitz do you know if `event_type` and `data_type` should be included here as well, or if they are internal (i.e. just used for tests) and should be omitted? Thank you!

Reply:
good question, @piokasar @maxhorowitz can you comment? I don't recall the history of those headers